### DISCUSSION TITLE
Are the options for bitrate and bandwidth deliberately synonyms?

### DISCUSSION THREAD

**Discussion by WatcherOfTheSkies** [esnet/iperf#1746] [Q&A] [answered]
Hi,

The client command line options --bitrate and --bandwidth both have the short-form of -b please can you clarify that this is deliberate?

The --bandwidth option does not appear in the help for 3.17.1

I'm just looking for clarification. Thank you.

**Comment by bmah888** [comment, marked as answer]
Yes, originally the application-level pacing was called `--bandwidth` (long option) or `-b` (short option). It was pointed out that the use of the term "bandwidth" is technically incorrect, because that's measure of the capacity of a channel, not a measure of what you sent through it. So we renamed the option `--bitrate`, but kept `--bandwidth` for backward compatibility. #583 has more details.
